### PR TITLE
ENH: add ability to read parameters from config file

### PR DESCRIPTION
**Pull request by justcalamari**
Solves #43

### CODE REVIEW

File: sampledb/reader.xsh [hunk 1]
```diff
@@ -62,31 +64,50 @@ def find_uids(uids, sdb):
...
-        uids = get_uid_from_qr()
-        _, unknown = find_uids(uids, sdb)
-        if len(unknown):
-            print('The following sample uids are not in the database:')
-            for uid in unknown:
-                print(uid)
-        with open('sample_schema.json') as sch:
-            schema = json.load(sch)
-        sdb.search(uid=list(uids)).download(filename, schema)
+    with open('sample_schema.json') as sch:
```
Comment (CJ-Wright) [inline sampledb/reader.xsh]: might want to do this with `package_data`

File: sampledb/data/sample_schema.json [hunk 2]
```diff
@@ -17,7 +17,6 @@
         "properties": {"type": "string"},
         "task_and_hazard_analysis": {"type": "string"},
         "project": {"type": "string"}
-},
+   },
    "order": ["uid", "sample_name", "phase_info", "sample-name_of_sample_background", "project", "sample_maker", "quantity", "disposition", "transport", "properties", "task_and_hazard_analysis"]
```
Comment (CJ-Wright) [inline sampledb/data/sample_schema.json]: Would it be possible to pull this below 80 characters per line

**Comment by justcalamari** [comment]
@CJ-Wright Let me know if there is anything else you want to do for the tests.

**Comment by CJ-Wright** [comment]
LGTM

**Pull request merged**